libs/tracing/ppx_tests/trace_debug.ml

```diff
@@ -0,0 +1,17 @@
+let top a b =
+  let add_one a = a + 1 [@@trace] in
+  add_one a + b
+
+let top a b =
+  let add_one a = a + 1 [@@trace_debug] in
```

Comment:
Would rather something like `[@@trace { level = Debug }]`. I think that is a more consistent design with other ppx and more extensible if there are more settings we'd like.